RWFramework/RWFramework/Playlist/Playlist.swift

```diff
@@ -236,7 +236,7 @@ extension Playlist {
             for it in data {
                 // TODO: Try to remove playlist dependency. Maybe pass into method?
                 it.playlist = self
-                it.player.renderingAlgorithm = .soundField
+                it.player.renderingAlgorithm = .HRTFHQ
```

Comment:
We need to remove this line entirely. It's unnecessary and messes up for me with `.HRTFHQ`.
```suggestion
```